packages/ui-heading/src/Heading/index.tsx

```diff
@@ -67,12 +83,26 @@ class Heading extends Component<HeadingProps> {
     }
   }
 
+  checkProps() {
+    const { variant, level, as } = this.props
+    if (variant) {
+      if (level) {
+        console.warn("Don't use 'level' with 'variant' ")
+      }
+      if (as) {
+        console.warn("Don't use 'as' with 'variant' ")
```

Comment:
I would add to the warning that this is coming from the `Heading` component

Reply:
Done